CHANGELOG.md

```diff
@@ -40,6 +40,7 @@ Format:
 -->
 ## [Unreleased]
 ### Added
+- added a test for multiline json values (only ones set by code not by user input)
```

Comment:
The "added" section usually has the specific meaning of "something added to the program interface", i.e. you as the user of this program can do something you couldn't before. This PR probably deserves it's own section, either "Internal" or "Testing", if it needs to be in the changelog at all.  Defering to @plocket, the changelog expert.

Reply:
I like the idea of adding `### Internal` to our headings! Thanks!

Reply:
Thanks for the input, and we discussed and implemented an internal section to the change log.